### PR TITLE
0xapi test

### PR DESCRIPTION
**Pull request by dxganta**
Just added a new test to test the working of the 0xapi

### CODE REVIEW

**Comment by gabrielpoca** [comment]
@dxganta shouldn't this be with the Liquity PR?

**Comment by dxganta** [comment]
> @dxganta shouldn't this be with the Liquity PR?

I guess so. But I used a separate branch to test since back then the Liquity Strategy was not compiling. So kept it that way.

File: test/0x/0xapi.fork.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    const amount = parseUnits('1', await sellToken.decimals());
+
+    await ForkHelpers.mintToken(sellToken, simpleTokenSwap.address, amount);
+
+    let url = `https://api.0x.org/swap/v1/quote?buyToken=${USDC}&sellToken=${sellToken.address}&sellAmount=${amount}`;
+
+    const { data, status } = await axios.get(url);
+
+    console.log(data);
```
Comment (0xrin1) [inline test/0x/0xapi.fork.spec.ts:52]: better without the console.log imo

**Comment by gabrielpoca** [comment]
@dxganta I'm still confused, what exactly are we testing here?

**Pull request closed without merging**